homework_12/src/components/main/Main.js

```diff
@@ -1,23 +1,52 @@
 import './main.scss';
-import Aside from '../aside/index';
-import Hello from '../hello/index';
-import List from '../list/index';
-import Generate from '../generate/index';
-import Counter from '../counter/index';
-import UserForm from '../form/index';
+import Button from '../button';
+import ShowText from '../showText';
+import ListUserPost from '../ListUserPost';
+import GeoLocation from '../geoLocation';
+import Stopwatch from '../stopwatch';
 
-class Main extends React.Component {
-  state = { users: [] }
+class Main extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      posts: []
+    };
+    this.getList();
+  }
+
+  getList = () => {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then(response => response.json())
+      .then(users => this.setState({ users }));
+  }
+
+  getPost = (user) => {
+    fetch('https://jsonplaceholder.typicode.com/posts')
```

Comment:
why do you fecth all possible posts? You need posts only for specific user, ie `https://jsonplaceholder.typicode.com/posts?userId=1`